src/web/screens/home.tsx

```diff
@@ -59,18 +82,51 @@ function Home() {
   return (
     <>
       <h1>Welcome back, {LoggedInUser?.profile.firstName}</h1>
+      <Suspense fallback={<Loading />}>
+        <AwaitTypesafe resolve={data.hasEmailContacts} errorElement={<AsyncErrorView />}>
+          {(hasEmailContacts) => (
+            <div>
+              {!hasEmailContacts && (
+                <Banner
+                  message='Please add your email contacts to stay up to date on all UID2 updates.'
+                  type='Info'
+                  fitContent
+                />
+              )}
+            </div>
+          )}
+        </AwaitTypesafe>
+      </Suspense>
       <div className='dashboard-cards-container'>
-        <Suspense fallback={<Loading />}>
-          <AwaitTypesafe resolve={data.counts} errorElement={<AsyncErrorView />}>
-            {(counts) => (
-              <SharingPermissionCard
-                sharingPermissionsCount={counts.sharingPermissionsCount}
-                bulkPermissionsCount={counts.bulkPermissionsCount}
-              />
-            )}
-          </AwaitTypesafe>
-        </Suspense>
-        <DocumentationCard />
+        <div>
+          <Suspense fallback={<Loading />}>
+            <AwaitTypesafe resolve={data.apiKeysToRotate} errorElement={<AsyncErrorView />}>
+              {(apiKeysToRotate) =>
+                apiKeysToRotate.length > 0 && (
+                  <div className='dashboard-cards'>
```

Comment:
The className is `dashboard-cards` but it only have seems to be used for an individual card - should we just call it `dashboard-card`? It'd also be nice if we didn't need this className - if you could find a way to do that? e.g. We already know these are cards within `dashboard-cards-container`.

Reply:
The reason I have both is to match the UX design where two cards are vertical ( `RotateApiKeysCard` and `SharingPermissionsCard`) and `DocumentationCard` is horizontal to those two.  I think we do need both the achieve this design, however I did remove some duplicated code between the two that isn't needed in the actual card ( `display: flex` and `gap: 22px`)